assets/cubyz/biomes/cave/crystal.zig.zon

```diff
@@ -1,6 +1,7 @@
 .{
 	.isCave = true,
 	.maxHeight = -512,
+	.minHeight = -44750,
```

Comment:
Honestly I kind of enjoyed the occasional crystal biomes down in the root world.
Would mind adding a copy of it specialized to be in the roots?